docs/getting-started/first-service.md

```diff
@@ -78,4 +78,20 @@ curl http://hello.default.127.0.0.1.nip.io
 Hello World!
 ```
 
+??? question "Are you seeing `curl: (6) Could not resolve host: hello.default.127.0.0.1.nip.io`?"
+
+    In some cases `*.nip.io` addresses are not able to be resolved. This can be fixed by having external nameservers resolve the addresses instead of the local nameserver.
```

Comment:
fwiw I'm a little confused by this. In what case should a local name server not work with nip.io? Is there a link to a GitHub issue or anything with more details about this?

Reply:
aha, saw the reference to https://github.com/deis/workflow/issues/833 in the previous version of this PR, so the problem is some nameservers block lookups that resolve to local (127.*) addresses? I think "local name server" vs external (and the "not able to be resolved" - they're configured not to resolve) might be what confused me here, since iiuc that's not the important factor.

Maybe we could clarify this a bit:

```suggestion
    In some cases your dns server may be set up not to resolve `*.nip.io` addresses. If you encounter this problem, it can be fixed by using a different nameserver to resolve these addresses. For example:
```

Reply:
@julz, nice - that sounds better, thanks.

Reply:
oops just realised the "For example" at the end doesn't fit with the next paragraph, sorry 🤦

Reply:
Removed :smile: